[PS-14487] PCP format information added (#199)
talkable/talkable-docs: source/advanced_features/passing_custom_data.rst

    The values you pass in the ``custom_properties`` have to be JSON Key-Value pairs themselves,

>     meaning that complex nested data structures can not be passed through.

Reviewer: Consistency is important.
```suggestion
    meaning that complex nested data structures cannot be passed through.
```

>     Property names (``eye_color``) can contain lowercase letters, numbers and ``_`` only and cannot begin with a number.

Reviewer: Let’s make it clear that `eye_color` is an example of a property.
```suggestion
    Property names (e.g. ``eye_color``) can contain lowercase letters, numbers and ``_`` only and cannot begin with a number.
```
